### PR TITLE
feat(basicfight): add conditionnal display

### PR DESCRIPTION
**Pull request by juliennicaise**

### CODE REVIEW

File: src/components/basicFight.jsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import Player from './player';
+import './basicFight.css';
 
 const weapons = ['rock', 'paper', 'scissors'];
```
Comment (rocambille) [inline src/components/basicFight.jsx:6]: files should be named after the components they declare, so this file should be Game.jsx ;)

**Review by rocambille — CHANGES_REQUESTED** [review]

File: src/components/basicFight.jsx (hunk 1 above)
Comment (juliennicaise) [reply src/components/basicFight.jsx:6]: change basicFight.css to  Game.css

Comment (rocambille) [reply src/components/basicFight.jsx:6]: you can do the same for basicFight.jsx -> Game.jsx

Comment (juliennicaise) [reply src/components/basicFight.jsx:6]: Done :)

**Pull request merged**